DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommanLineTool.UnitTests/AnonymizerToolUnitTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using System.Threading.Tasks;
+using Dicom;
+using Microsoft.Health.Dicom.Anonymizer.Core.Tool;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.UnitTests
+{
+    public class AnonymizerToolUnitTests
+    {
+        [Fact]
+        public async Task GivenOneDicomFile_WhenAnonymize_ResultWillBeReturnedAsync()
+        {
+            var commands = "-i DicomFiles/I341.dcm -o I341.dcm";
+            await AnonymizerCliTool.Main(commands.Split());
+            var dicomFile = DicomFile.Open("I341.dcm");
+            var expectedDicomFile = DicomFile.Open("DicomResults/I341.dcm");
```

Comment:
I am not recommending to access real files in tests, especailly write/delete them. If you are running the tests on build server, you may not have the wirte access so your tests will never pass. Try to use mock tools or simply pass a simple file content in string to do the test.

Reply:
I can refine tests in next PR.